modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -49,30 +50,32 @@ namespace {{packageName}}.Client
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ApiClient" /> class
-        /// with default configuration and base path ({{{basePath}}}).
+        /// with default configuration and base path (<see cref="SERVER_DEFAULT_ADDRESS"/>).
         /// </summary>
-        public ApiClient()
-        {
+        public ApiClient() {
+            log.Trace("start");
```

Comment:
Actually, I don't think so. As far as I know this is a common practice to implement your logging with pulling method name from the logger static object, not from the stack. For example, NLog specification with static declaration https://github.com/NLog/NLog/wiki/Tutorial#creating-loggers